compiler/bitproto/renderer/impls/py/renderer.py

```diff
@@ -289,6 +344,20 @@ def before(self) -> None:
         self.push_definition_docstring(indent=4)
 
 
+class BlockMessageDictFactory(BlockMessageBase, BlockWrapper[F]):
+    def wraps(self) -> Block[F]:
+        pass
+
+    @override(BlockWrapper)
+    def before(self) -> None:
+        self.push("@staticmethod")
+        self.push("def dict_factory(kv_pairs):")
+        self.push(
+            f"    return {{k: v for k, v in kv_pairs if not k.startswith('{_enum_field_proxy_prefix}')}}"
+        )
+        self.push_definition_docstring(indent=4)
```

Comment:
>         self.push_definition_docstring(indent=4)

Hmm, should this line be removed?

Reply:
Oh, yes, indeed, typical copy-paste artifact... I've removed it and pushed the commit to the branch.